### ISSUE TITLE
Upgrade VS Code Camel K extension to version 0.0.13

### ISSUE THREAD

**Issue by apupier** [eclipse-che/che#16435]
### Is your enhancement related to a problem? Please describe.

upgrade to a newer version.

### Additional context

- Camel K runtime used has been updated, so [sidecar](https://github.com/che-dockerfiles/che-sidecar-camelk) needs to be updated too to use Camel K runtime 1.0.0-RC2
- as a second iteration, there is now an [option](https://github.com/camel-tooling/vscode-camelk/blob/ad018b94157ed3eff383c487758661c35847d0c2/package.json#L81) to specify which Camel K runtime t use. It allows to stick with a specific version even when there is a new one released. it woudl be useful to leverage that in Che environment to keep using the version in the sidecar. [example](https://github.com/eclipse/che-devfile-registry/pull/216/files) setting a preference in devfile

the vsix is available here: https://download.jboss.org/jbosstools/vscode/stable/vscode-camelk/vscode-camelk-0.0.13-403.vsix

see also https://issues.redhat.com/browse/FUSETOOLS2-233

**Comment by apupier** [comment]
prepared branch https://github.com/apupier/che-plugin-registry/tree/16435-CamelK-0.0.13 but will require problem with credentials to publish sidecars from CI to be solved, see https://github.com/apupier/che-sidecar-camelk/runs/531035794?check_suite_focus=true

**Comment by ericwill** [comment]
> prepared branch https://github.com/apupier/che-plugin-registry/tree/16435-CamelK-0.0.13 but will require to be updated with correct sha when problem with credentials to publish sidecars from CI to be solved, see https://github.com/apupier/che-sidecar-camelk/runs/531035794?check_suite_focus=true

I think you have to push to your own repo for now, once the PR is merged in che-sidecar-camelk then it will automatically be pushed to the Eclipse quay.io repo. It's probably failing because your fork doesn't have the correct quay.io secret (for the Eclipse repos) attached to it.

**Comment by apupier** [comment]
> I think you have to push to your own repo for now, once the PR is merged in che-sidecar-camelk then it will automatically be pushed to the Eclipse quay.io repo.

the 0.0.13 branch has been already pushed https://github.com/che-dockerfiles/che-sidecar-camelk/tree/0.0.13
The mentioned failing build is from this pushed branch

**Comment by apupier** [comment]
my bad.
So I pointed to a build of the branch which was on my repository. (i wasn't expecting a build from branches on my own repository). Not yet familiar with GitHub Actions (and also I should have think a bit more)
The build from the che organization is working fine https://github.com/che-dockerfiles/che-sidecar-camelk/actions/runs/62350912

**Comment by apupier** [comment]
firs iteration done.

Will open another issue if/when we tackle the second one:

`as a second iteration, there is now an option to specify which Camel K runtime t use. It allows to stick with a specific version even when there is a new one released. it woudl be useful to leverage that in Che environment to keep using the version in the sidecar. example setting a preference in devfile`
